src/redux-store.ts

```diff
@@ -9,11 +9,7 @@ import WebSocketClient from "./websocket-client";
 
 const { createReduxHistory, routerMiddleware, routerReducer } =
   createReduxHistoryContext({
-    history: createBrowserHistory({
-      basename: `${import.meta.env.VITE_APP_BASENAME}${
-        import.meta.env.VITE_APP_VITE_BASENAME
-      }`,
-    }),
+    history: createBrowserHistory(),
```

Comment:
I'm assuming this change is because `basename` was a react-router v5 fix that's no longer required?

Reply:
Indeed! Basename is now specified as a Router prop.